vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -382,16 +399,16 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
                 member this.GetMostRecentLines(n:int) : string[] =
                     GetToolWindowAsITestVFSI().GetMostRecentLines(n)
             
+            // Can we get rid of this now?
```

Comment:
If we're not calling anything here, then yes, this can go. Same with the registration for this IOleComponent